src/server/game/Entities/Player/Player.cpp

```diff
@@ -4164,8 +4164,11 @@ void Player::BuildPlayerRepop()
     packet.PlayerGUID = GetGUID();
     GetSession()->SendPacket(packet.Write());
 
-    if (getRace() == RACE_NIGHTELF)
+    // If the player has the Wisp racial then cast the Wisp aura on them
+    if (HasSpell(20585))
```

Comment:
magic number

Reply:
so are both spell casts below...

Reply:
Is it really needed to have enums for every spell, considering they're not exactly "magic numbers" as it's evident that they're spells and there are various lookup tools for them?

Reply:
Yes, because you save that lookup time when using enums. And you have to remind which spellid is related to which spell when reading longer code. With enums you directly know whats happening.